README.md

```diff
@@ -16,13 +16,19 @@ Configuring `winrepo` is not handled by this formula.
 
 Installs the Nessus Agent.
 
-## Windows Configuration
+## Configurations
```

Comment:
I think it's fine without this section... Prefer not to have 4-deep headings...

Reply:
Okay.  I'll remove.